unit_tests/Test_LocalDeepCopy.cpp

```diff
@@ -123,8 +124,10 @@ void test_localdeepcopy(int i1,
                 v_R(my_rank,j) = 0x123;
               });
       
+      //Run 1 Team only.
       Kokkos::single(Kokkos::PerTeam(team), [&]() {
-        Kokkos::Experimental::local_deep_copy(team, v_R_cpy, v_R);
+        Kokkos::Experimental::local_deep_copy(v_R_cpy, v_R);
+        //Kokkos::Experimental::local_deep_copy(team, v_R_cpy, v_R); //Currently not working. TBD
```

Comment:
What isn't working? I'm a bit uncomfortable committing this.

Reply:
The overload of local_deep_copy that takes the team parameter performs a local deep copy using a parallel_for over a flattened iteration space and reconstructs the indexes from a flat index i, shown [here](https://github.com/kokkos/kokkos/blob/03ded4e816da5927b946065b005ebfdf38e5a438/core/src/Kokkos_CopyViews.hpp#L1676). The reconstruction is incorrect for our case. Making this work requires to specialize local_deep_copy to a remote space.